### DISCUSSION TITLE
Availability for the ESMValTool workshop at the Met Office in autumn 2022

### DISCUSSION THREAD

**Discussion by mo-tgeddes** [ESMValGroup/Community#39] [Meetings]
Hi @ESMValGroup/esmvaltool-developmentteam,
Could you please fill in this form for what dates you would be available for a workshop? Format will be hybrid: in-person attendance at Met Office (Exeter, UK) encouraged for those that can travel, plus remote participation by MS Teams.

https://forms.office.com/r/M4ytWg6T3p 

The form will close at 8pm(BST) on Friday 29th July.

Thanks,
Theo Geddes
Met Office

**Comment by zklaus** [comment]
Thanks, @mo-tgeddes, for organizing this. I started filling in the form, but I think a few dates got mixed up. The second slot would be Tue, Oct 18--Thu, Oct **20** (not 21), the corresponding technical session would be on Fri, Oct **21** (not 22) and I guess the third slot is intended to be Tue, Nov **29** (not 28)--Thu, Dec 1, with an associated technical session on Mon, Nov **28** (not 27).

I will also note that the printing option didn't work very well for me, with several options missing or not reflected correctly in the printout, though obviously that is not your fault at all, just intended as feedback on the tool.

**Reply by mo-tgeddes** [reply]
Hi, 
Thank you for pointing that out. I have fixed it now.

**Comment by mo-tgeddes** [comment]
Hi, 
Just a reminder that the form is still open, but will close this Friday at 8pm BST.

Thanks,
Theo Geddes

**Comment by mo-tgeddes** [comment]
Hi @ESMValGroup/esmvaltool-developmentteam, 
the dates for the Autumn workshop will be 18th - 20th October (Tuesday - Thursday).
